src/models/customerSchema.js

```diff
@@ -0,0 +1,39 @@
+
+const mongoose = require("mongoose");
+
+const customerSchema = new mongoose.Schema({
+  customer_id: {
```

Comment:
Since MongoDB created an id for each input we do not need to have a customer id, I know that we have a customer_id in database design but in here we do need to have such a thing.